### PR TITLE
fix: set resources of minio from spec of harbor cluster

### PR DESCRIPTION
**Pull request by heww**
Signed-off-by: He Weiwei <hweiwei@vmware.com>

### CODE REVIEW

File: pkg/cluster/controllers/storage/tenant.go [hunk 1]
```diff
@@ -215,7 +215,7 @@ func (m *MinIOController) generateMinIOCR(ctx context.Context, harborcluster *go
 					Servers:             harborcluster.Spec.InClusterStorage.MinIOSpec.Replicas,
 					VolumesPerServer:    harborcluster.Spec.InClusterStorage.MinIOSpec.VolumesPerServer,
 					VolumeClaimTemplate: m.getVolumeClaimTemplate(harborcluster),
-					Resources:           *m.getResourceRequirements(harborcluster),
+					Resources:           harborcluster.Spec.InClusterStorage.MinIOSpec.Resources,
```
Comment (Ghostbaby) [inline pkg/cluster/controllers/storage/tenant.go:218]: we will not define default resource?

**Review by soulseen — APPROVED** [review]
/lgtm

File: manifests/samples/harbor_full_minio.yaml [hunk 2]
```diff
@@ -130,19 +130,11 @@ spec:
       volumesPerServer: 2
       volumeClaimTemplate:
         spec:
-          storageClassName: standard
```
Comment (bitsf) [inline manifests/samples/harbor_full_minio.yaml:133]: should also change in harbor_core_minio.yaml

File: pkg/cluster/controllers/storage/tenant.go (hunk 1 above)
Comment (heww) [reply pkg/cluster/controllers/storage/tenant.go:218]: Why do we need to define the default resource for the minio pod when the customers don't want the limited resources used by it?

**Review by steven-zou — APPROVED** [review]
LGTM

**Review by holyhope — APPROVED** [review]

**Pull request merged**